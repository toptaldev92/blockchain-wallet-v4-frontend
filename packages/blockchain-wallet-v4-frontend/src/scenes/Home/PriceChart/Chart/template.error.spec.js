import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import TemplateError from './template.error'
jest.mock('blockchain-info-components', () => ({ Text: 'text' }))

describe('PriceChart template error', () => {
  it('renders correctly', () => {
    const baseProps = { children: 'Error message' }
    const component = shallow(<TemplateError {...baseProps} />)
    const tree = toJson(component)
    expect(tree).toMatchSnapshot()
  })
})
