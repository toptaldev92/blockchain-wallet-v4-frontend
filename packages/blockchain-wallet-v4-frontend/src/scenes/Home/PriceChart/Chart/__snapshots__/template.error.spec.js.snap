// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chart template error renders correctly 1`] = `
<templateerror__Wrapper>
  <text
    color="red"
    size="12px"
    weight={300}
  >
    Error message
  </text>
</templateerror__Wrapper>
`;
